src/pymovements/gaze/experiment.py

```diff
@@ -171,3 +248,14 @@ def pos2vel(
         return transforms_numpy.pos2vel(
             arr=arr, sampling_rate=self.sampling_rate, method=method, **kwargs,
         )
+
+    def __str__(self: Any) -> str:
+        """Print experiment."""
+
+        def shorten(value: Any) -> str:
+            if isinstance(value, float):
+                value = f'{value:.2f}'
+            return value
+
+        attributes = ', '.join(f'{key}={shorten(value)}' for key, value in vars(self).items())
```

Comment:
this will probably need something like `if not key.startswith('_')` to not list any private attributes with a leading underscore